src/com/amazon/ion/benchmark/GeneratorOptionsValidator.java

```diff
@@ -53,6 +53,12 @@ public static void checkValid(String [] args, Map <String, Object> optionsMap) {
                 case DECIMAL:
                     GeneratorOptionsValidator.throwException(INVALID_FOR_DECIMAL, commandLine);
                     break;
+                case INT:
+                case FLOAT:
+                case BLOB:
+                case CLOB:
+                case SYMBOL:
+                    break;
```

Comment:
Does this mean we silently do nothing for these types?

Reply:
> Does this mean we silently do nothing for these types?

Thanks for catching this! For these types there should be one step to check if the invalid options provided. I'll add it in the next commit.